pandas/compat/_optional.py

```diff
@@ -8,8 +8,9 @@
 VERSIONS = {
     "bs4": "4.6.0",
     "bottleneck": "1.2.1",
+    "fsspec": "0.7.4",
     "fastparquet": "0.3.2",
-    "gcsfs": "0.2.2",
+    "gcsfs": "0.6.0",
```

Comment:
Will need to update these in the "Increased minimum versions for dependencies" section of the whatsnew.